### PR TITLE
Fix accent chars

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: lib/theme-options.php [hunk 1]
```diff
@@ -119,6 +119,31 @@ function add_options_page_metabox() {
 			'type' => 'checkbox',
 		) );
 
+		// SPLASH
+
+		$options_metabox->add_field( array(
```
Comment (mixcas) [inline lib/theme-options.php:124]: where is this used?

Comment (varffs) [inline lib/theme-options.php:124]: on the splash! in partials/splash.php

Comment (mixcas) [inline lib/theme-options.php:124]: i don't see it, thats why i'm asking

Comment (mixcas) [inline lib/theme-options.php:124]: im talking about `_igv_splash_title`

File: partials/splash.php [hunk 2]
```diff
@@ -7,11 +7,21 @@
...
-      <h3 class="font-larger font-bold font-uppercase">CASA LUIS BARRAGÁN</h3>
+      <h3 class="font-larger font-bold font-uppercase">CASA LUIS BARRAGÁN</h3>
     </div>
 
     <div id="splash-text" class="font-huge font-leading-zero u-flex-center">
-      <?php echo __('[:es]es una plataforma cultural y artística auspiciada por Casa Luis Barragán con el apoyo de Colección FEMSA.[:en]a cultural and artistic platform hosted by Casa Luis Barragán with the support of Colección FEMSA.');
+      <?php
+
+        if (qtranxf_getLanguage() === 'es') {
+          $splash_text = IGV_get_option('_igv_splash_text');
```
Comment (varffs) [inline partials/splash.php:17]: its here?

File: lib/theme-options.php (hunk 1 above)
Comment (varffs) [inline lib/theme-options.php:124]: o this is the title thing. its like a title for the metabox display not an actual field

**Pull request merged**